### ISSUE TITLE
Is SharePoint Starter Kit ready for Production use?

### ISSUE THREAD

**Issue by rajeshb4** [SharePoint/sp-starter-kit#71]
#### Category
- [x] Question
- [ ] Bug
- [ ] Enhancement

I understand that the starter kit works in targeted release tenant only. We showcased the sample implementation of sharepoint starter kit to our client and now they want it implemented in production tenant. 

Is SharePoint starter kit as of its current state recommended for production tenants? Please clarify.

**Comment by eoverfield** [comment]
The SP Starter Kit requires the entire tenant be configured for targeted release for all users. I would never recommend a production tenant be configured for targeted release.

That does not mean you cannot provision most of the starter kit to a non-targeted release tenant though. You can, you will have to work through a few errors and warnings during installation, and certain SPFx webparts that require the GraphClient will not function at this time. Hopefully soon the targeted release requirement will not be needed although there is no official timeline yet.

That all being said, the starter kit is a community driven effort, meaning, although the implementation strategy and code is solid in my opinion, it is not officially supported or maintained by Microsoft or any other third party vendor. The starter kit was definitely designed to help you kickstart your Office 365 / SPO journey, showing off what is possible, as well as delivery solutions now.

You should expect more changes to the kit throughout the year and beyond. If you were to use the kit in production, I would recommend you review all components you intend to use including the provisioning, site scripts, site designs, webparts, and extensions so that you are confident you know what is there, and you could fix any issue that may arise. If you are at that point, I see no reason why would couldn't use part or all of the current kit in a production tenant.

**Comment by nicholas-morris** [comment]
I saw Beau Cameron has a version for standard release mentioned here: https://beaucameron.net/2018/06/25/pnp-starter-kit-for-standard-release-tenants/

**Comment by bcameron1231** [comment]
As @nicholas-morris  mentioned, I did post a version of it for tenants that aren't in First-Release. But please please be aware, that some bugs still exist in the PnP Starter Kit and be cognizant of where you deploy this, and I also won't be maintaining that repo with any new fixes.

**Comment by mag210** [comment]
Hi all, do we have any updates on when the targeted release requirement will no longer be needed for the starter kit? Like many, I've found the kit to be a brilliant way to sell the benefits of 365 and show the "art of the possible" to the institution. I would really like to use the kit as a basis for our intranet development but we can't really do this until it supports standard release. I guess the main obstacle are the web parts using the new Graph client.... Any updates would be appreciated.
-Max

**Comment by eoverfield** [comment]
@mag210 The targeted release requirement is due to the utilization of the GraphClient which has not yet gone GA in SPO itself. Based on recent PnP Community calls, we are expecting SPFx v 1.6 in the next few weeks or early Q4. We have been told that v1.6 is expected to include the GA of the GraphClient.

Also consider that Ignite 2018 is later this month (Sept 2018). I have to believe that is a big driver to release 1.6 either right before or right around Ignite although that is only speculation.

**Comment by mag210** [comment]
Hi @eoverfield , I see that the 1.6 version of SPFX is now in general release. Does this mean we can now use the starter kit in production tenancies which are not setup for First Release? The Graph APIs look like they are now available for GA so I would assume so?
Thanks, 
-Max

**Comment by mag210** [comment]
Thanks @eoverfield. Looks like we're not too far off. This is going to be a huge help for SPFX Intranet developers all over...

**Comment by eoverfield** [comment]
@mag210 Yes, once @waldekmastykarz's commit is merged into the master branch, we should all be set and all tenants should be able to use the starter kit without the targeted release requirement. If you want to test the updates now, pull the dev branch and deploy that to one of your dev tenants.

**Comment by Rtoribiog** [comment]
hi guys any update on this ?  i want to test it in production tenant. Thanks

**Comment by eoverfield** [comment]
@Rtoribiog This is related to https://github.com/SharePoint/sp-starter-kit/issues/168#issuecomment-427535039. I am working to get us an updated timeline.

**Comment by pallinixevo** [comment]
Dear All,

looking on https://provisioning.sharepointpnp.com/  Targeted release option is requested despite 1.8.x SPFX version is available. could u kindly update?

**Comment by bcameron1231** [comment]
Closing - This has been answered on this thread. If you have any further questions,  please open a new issue.
